docs/application/native/tutorials/details/app-filtering.md

```diff
@@ -317,6 +317,7 @@ package:
 | `http://tizen.org/feature/shortcut`      | Specify this key, if the application requires the shortcut feature. | 4.0   |
 | `http://tizen.org/feature/sip.voip`      | Specify this key, if the application requires the Voice Over Internet Protocol (VOIP) feature. | 2.2.1 |
 | `http://tizen.org/feature/speech.control` | Specify this key, if the application requires the voice control feature. | 2.4   |
+| `http://tizen.org/feature/speech.control_manager` | Specify this key, if the application requires the voice control manager feature. | 5.5   |
```

Comment:
Question. Doesn't wearable also support this feature?

In app-filtering.md, we have two tables each for mobile and wearable.
If wearable also supports speech.control_manager feature, please add this feature to the table for wearable, too.

Reply:
hello, @mijin85cho 
Good question to ask.
I missing wearable table. so, I will added thank you :smile: